hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileAppendTest.java

```diff
@@ -80,16 +87,31 @@ class FileAppendTest extends FileTestBase {
     @Mock(strictness = Mock.Strictness.LENIENT)
     protected Account payerAccount;
 
+    @Mock(strictness = Mock.Strictness.LENIENT)
+    HederaFileNumbers fileNumbers;
+
+    @Mock(strictness = Mock.Strictness.LENIENT)
+    FeeCalculator feeCalculator;
+
+    @Mock(strictness = Mock.Strictness.LENIENT)
+    private FeeAccumulator feeAccumulator;
+
     protected Configuration testConfig;
 
     private FileAppendHandler subject;
 
     @BeforeEach
     void setUp() {
-        subject = new FileAppendHandler();
+        subject = new FileAppendHandler(fileNumbers);
         testConfig = HederaTestConfigBuilder.createConfig();
-        lenient().when(preHandleContext.configuration()).thenReturn(testConfig);
-        lenient().when(handleContext.configuration()).thenReturn(testConfig);
+        when(preHandleContext.configuration()).thenReturn(testConfig);
+        when(handleContext.configuration()).thenReturn(testConfig);
+        when(handleContext.feeCalculator(any(SubType.class))).thenReturn(feeCalculator);
+        when(feeCalculator.calculate()).thenReturn(Fees.FREE);
+        when(feeCalculator.addBytesPerTransaction(anyLong())).thenReturn(feeCalculator);
+        when(feeCalculator.addStorageBytesHours(anyLong())).thenReturn(feeCalculator);
+        when(handleContext.feeAccumulator()).thenReturn(feeAccumulator);
+        when(feeCalculator.legacyCalculate(any())).thenReturn(Fees.FREE);
```

Comment:
Instead of mocking these, could we have used the FakeFeeCalculator and FakeFeeAccumulator from app-spi test fixtures?

Reply:
I tried originally I think, and it didn't work for me. I don't remember why I can try again.
I see we use the same in Consensus

Reply:
The fake versions might not have all the methods implemented; so the objects might need to be updated, but I suspect that making those work for the general case might be more helpful than mocking in each service.